src/utils.ts

```diff
@@ -7,3 +7,14 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPartItems(
+  items: string[],
+  currentPage: number,
+  pageSize: number,
+) {
```

Comment:
The function `getPartItems` would benefit from a return type annotation for better type checking and readability. You can specify the return type as `string[]` after the parameter list.